SPECS/ldns/ldns.spec

```diff
@@ -32,48 +32,39 @@
 
 Summary:        Low-level DNS(SEC) library with API
 Name:           ldns
-Version:        1.7.0
-Release:        32%{?dist}
-License:        BSD
+Version:        1.8.3
+Release:        1%{?dist}
+License:        BSD-3-Clause
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Distribution:   Azure Linux
```

Comment:
I completely forgot about the `Distribution` tag. Do we have a separate task to update these in bulk?